src/index.ts

```diff
@@ -1,17 +1,26 @@
-import {Agenda, Person, Pet} from './agenda';
+import {Agenda} from './agenda';
 
+// Create agenda
 let agenda = new Agenda();
 
-let person = new Person ('Vesal', 'Fayazi', 'Regeringsgatan 28');
-let pet = new Pet ('Rocky', 'Rabbit', 'Stallone');
+// Load file
+agenda.deserialize('../contacts.json');
```

Comment:
If the file contains contacts, you do not add them manually as the lines below.
Otherwise, you add them manually as you did.